force-app/main/default/classes/Rollbar.cls

```diff
@@ -29,7 +29,11 @@ global with sharing class Rollbar {
     }
 
     public static Rollbar init(String accessToken, String environment) {
-        return Rollbar.init(new Config(accessToken, environment));
+        if (accessToken == null || accessToken.length() == 0 || accessToken == '') {
```

Comment:
this is fine, but maybe something like this to make it more readable? just a suggestion


var no_access_token = (accessToken == null || accessToken.length() == 0 || accessToken == ‘’);

if (no_access_token) {
	return Rollbar.init(new Config());
} else {
   return Rollbar.init(new Config(accessToken, environment));
}